src/pages-and-resources/discussions/app-config-form/apps/shared/blackout-dates/BlackoutDatesInput.jsx

```diff
@@ -0,0 +1,83 @@
+import React, { useState } from 'react';
+import { Form } from '@edx/paragon';
+import { useFormikContext, getIn } from 'formik';
+import PropTypes from 'prop-types';
+
+import FormikFieldFeedback from '../../../../../../generic/FormikFieldFeedback';
+
+const BlackoutDatesInput = ({
+  index,
+  value,
+  type,
+  label,
+  fieldName,
+  helpText,
+  fieldClasses,
+  feedbackClasses,
+  formGroupClasses,
+}) => {
+  const {
+    handleChange, handleBlur, errors, touched,
+  } = useFormikContext();
+  const [inFocus, setInFocus] = useState(false);
+  const fieldError = getIn(errors, `blackoutDates[${index}].${fieldName}`);
+  const fieldTouched = getIn(touched, `blackoutDates[${index}].${fieldName}`);
+  const isInvalidInput = Boolean(!inFocus && fieldError && fieldTouched);
```

Comment:
why we ignoring errors if field is not in focus. --> checking this point.

Reply:
@mehaknasir did you check this point. is it clear or not?
we display input helper text when a field is focused.  We display an error message when the field is out of focus(onBlur) and has an error.

Reply:
yes thanks.